net-libs/biblesync/biblesync-2.1.0-r1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="A multicast protocol to support Bible software shared co-navigation"
+HOMEPAGE="https://wiki.crosswire.org/BibleSync"
+SRC_URI="https://github.com/karlkleinpaste/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="static"
+
+PATCHES=( "${FILESDIR}"/${P}-cmake4.patch )
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_SHARED_LIBS=$(usex !static)
```

Comment:
Please drop IUSE static. The reason given for it in bug https://bugs.gentoo.org/623808 does not appear to be true for current app-text/xiphos anymore, especially since that went on to actually force `-static` afterwards.

So it should never have been allowed to build static in the first place. `cmake.eclass` sets `BUILD_SHARED_LIBS=ON` unconditionally so the cmake arg can just be dropped.

Reply:
Very good catch. I have changed it.